subprojects/docs/src/samples/userguide/files/copy/kotlin/build.gradle.kts

```diff
@@ -2,6 +2,7 @@
 import org.apache.tools.ant.filters.FixCrLfFilter
 import org.apache.tools.ant.filters.ReplaceTokens
 // end::filter-files[]
+import java.util.concurrent.Callable
```

Comment:
This isn't required anymore correct?

Reply:
It is still required on `gradle/gradle` at this point